### DISCUSSION TITLE
Hover states break in componentized elements

### DISCUSSION THREAD

**Discussion by tobyab** [tailwindlabs/tailwindcss#11902] [Help]
**What version of Tailwind CSS are you using?**

v3.3.3

**What build tool (or framework if it abstracts the build tool) are you using?**
Autoprefixer 10.4.15, Next.js 13.4.12, PostCSS 8.4.28

**What version of Node.js are you using?**
v20.5.1

**What browser are you using?**
Arc Version 1.5.0 (41058) - (Chromium Engine Version 116.0.5845.110)

**What operating system are you using?**
macOS

**Reproduction URL**
Example of a componentized button function that's hover state breaks when imported and used elsewhere https://github.com/developedbytoby/website/blob/dcf09a011cec27726b7a8375b4edc04fc762bf11/components/design/button.tsx#L4

**Describe your issue**
When I componentize an element which has Tailwind styling, and use that element in another place, the hover state breaks. Please let me know if you need me to elaborate on anything! :)

**Comment by Jacklaw12** [comment]
This could be due to class scoping or conflicts in your CSS. Ensure that your component styles are scoped correctly and that your Tailwind CSS classes are being applied consistently.  [YourTexasBenefits App](https://www.yourtexasbenefits.bid/)
